docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -96,9 +98,121 @@ PIDFile=/run/user/1000/overlay-containers/ccfd5c71a088768774ca7bd05888d55cc28769
 WantedBy=multi-user.target default.target
 ```
 
+Creating systemd unit file for a pod with `--new` flag
```

Comment:
Can we make this a new sub-section with `###`? It will help finding it when skimming the man page.